packages/plugin-github-actions/index.js

```diff
@@ -8,6 +9,21 @@ export default {
       return '';
     }
 
+    if (target.startsWith('.')) {
+      const [, org, repo, , sha] = path.split('/');
+
+      return `{BASE_URL}/${join(org, repo, 'tree', sha, target)}`;
+    }
+
+    const [org, repo, ...folder] = target.split('/');
+
+    if (folder.length) {
+      return [
+        `{BASE_URL}/${join(org, repo, 'tree', 'main', ...folder)}`,
```

Comment:
This is because GitHub is starting to rename their default branches from `master` to `main` and I think this is supposed to become the site's default for new repos soon.

Reply:
Do you mind if we address this separately? There are a few places where master is hardcoded and I think this needs some additional thoughts

Reply:
Without this the test won't pass because the [github/codeql-action](https://github.com/github/codeql-action) repo uses `main` now.

Reply:
😄  Ahh I see, thanks for the hint.

Reply:
Opened #974 for this.